hack/env.sh

```diff
@@ -17,6 +17,7 @@
 # Environment variables used in development. This file is sourced in the other scripts.
 
 # General vars
+export ARGOCD_BUILD_ARGS="--go-build-args -tags=openshift"
```

Comment:
This basically means that every build would include the openshift tags?

Reply:
This is still an issue here?

Reply:
I don't think we should do this here, as it means that the environment will always build for openshift.